### PR TITLE
Specs: more controller and feature tests

### PR DESCRIPTION
**Pull request by davidwessman**
Tests:
- VoteUser
- Vote
- Sign up, sign in, confirm account

Changing size of header-tags

### CODE REVIEW

File: app/controllers/vote_posts_controller.rb [hunk 1]
```diff
@@ -16,10 +16,12 @@ def create
     @vote_post.user = current_user
 
     if VoteService.user_vote(@vote_post)
-      redirect_to votes_path, notice: alert_create(VotePost)
+      flash[:notice] = alert_create(VotePost)
     else
-      render :new, status: 422
+      flash[:alert] = I18n.t('vote_post.already_voted')
```
Comment (navro) [inline app/controllers/vote_posts_controller.rb]: Is this always the case when VoteService returns false? What happens with other errors?

**Comment by navro** [comment]
Good work! +1 except for the comment above :)

File: app/controllers/vote_posts_controller.rb (hunk 1 above)
Comment (davidwessman) [inline app/controllers/vote_posts_controller.rb]: Definitely mistaken here. Will fix :smile: 

I guess the error should be added to one of the fields instead.

**Comment by navro** [comment]
Looks good now!

**Pull request merged**